src/validation.h

```diff
@@ -118,6 +118,7 @@ static const int64_t MAX_FEE_ESTIMATION_TIP_AGE = 3 * 60 * 60;
 static const bool DEFAULT_PERMIT_BAREMULTISIG = true;
 static const bool DEFAULT_CHECKPOINTS_ENABLED = true;
 static const bool DEFAULT_TXINDEX = false;
+static const char* const DEFAULT_BLOCKFILTERINDEX = "0";
```

Comment:
I believe we make this `extern` in the header, so that it is not added to each obj file?

Reply:
Yeah. This applies to all the defaults, right?

Reply:
I think only strings and cstrings and not POD types